### DISCUSSION TITLE
Is withResponsiveImages  not available on syncFromMediaLibraryRequest?

### DISCUSSION THREAD

**Discussion by Braunson** [spatie/laravel-medialibrary#2913] [Q&A]
Is `withResponsiveImages` not available on `syncFromMediaLibraryRequest` or `addFromMediaLibraryRequest`?

I want to make the original uploaded images responsive however when doing this in my Livewire component it throws an error regarding the `withResponsiveImages` method not available.

```php
if ($this->editing->wasRecentlyCreated) {
    $this
        ->editing
        ->syncFromMediaLibraryRequest($this->images)
        ->withResponsiveImages()
        ->toMediaCollection('images');
} else {
    $this->editing
        ->addFromMediaLibraryRequest($this->images)
        ->withResponsiveImages()
        ->toMediaCollection('images');
}
```

**Comment by ChrisThompsonTLDR** [comment]
I don't have an answer, but am following because I'm experiencing the same issue.
